stdlib/@python2/__builtin__.pyi

```diff
@@ -861,7 +861,7 @@ def filter(__function: None, __iterable: Iterable[Optional[_T]]) -> List[_T]: ..
 @overload
 def filter(__function: Callable[[_T], Any], __iterable: Iterable[_T]) -> List[_T]: ...
 def format(__value: object, __format_spec: str = ...) -> str: ...  # TODO unicode
-def getattr(__o: Any, name: Text, __default: Any = ...) -> Any: ...
+def getattr(__o: Any, name: Text, __default: _T = ...) -> Union[Any, _T]: ...
```

Comment:
I believe this needs to be done with overloads for two-argument and three-argument `getattr`, though let's see what CI says.

Reply:
yeah, I figured I'd wait for CI. I was surprised when I didn't find any overloads already defined, so I'd assume that this would work, given that what's on master presumably works.

Reply:
Well, the reason I'm suspicious about it is the typevar + default combination. I think mypy doesn't really know what to do with `_T` if the argument isn't passed. But if it falls back to Any, I suppose that's fine.